apps/financial-aid/web-osk/pages/index.tsx

```diff
@@ -1,34 +1,20 @@
-import React, { useContext, useEffect } from 'react'
-import { Button, Text } from '@island.is/island-ui/core'
-
+import { Routes } from '@island.is/financial-aid/shared/lib'
 import { useRouter } from 'next/router'
-import { ContentContainer } from '@island.is/financial-aid-web/osk/src/components'
+import { useContext, useEffect } from 'react'
+import { UserContext } from '../src/components/UserProvider/UserProvider'
 
 const Index = () => {
+  const { isAuthenticated } = useContext(UserContext)
   const router = useRouter()
   useEffect(() => {
     document.title = 'Umsókn um fjárhagsaðstoð'
   }, [])
 
-  return (
-    <ContentContainer>
-      <Text as="h1" variant="h2" marginBottom={2}>
-        Abbabba þú hefur villst
-      </Text>
+  if (isAuthenticated) {
+    router.push(Routes.application)
+  }
 
-      <Button
-        onClick={() => {
-          router.push('/umsokn')
-        }}
-        data-testid="logout-button"
-        preTextIconType="filled"
-        type="button"
-        variant="primary"
-      >
-        heim
-      </Button>
-    </ContentContainer>
-  )
+  return null
```

Comment:
What is this component used for?

Reply:
In Osk, we want to have our application domain under `/umsokn` and there is going to be a landing page under `/` which has not been implemented so for now this page is only used to direct the user into the login flow, with a redirect url of `/umsokn`. And then later we will add the landing page :)